java.net.ConnectException: Connection refused (Connection refused)

Hello @jrhenderson1988,

Your blog [Building Microservices with Spring Boot - Part 1](https://blog.scottlogic.com/2019/10/31/building-microservices-with-spring-boot) was well written, concise and informative. 

I did have a problem with running this sample locally

> 2019-11-09 09:17:11.875 ERROR 10611 --- [et_localhost-17] c.n.e.cluster.ReplicationTaskProcessor   : Network level connection to peer localhost; retrying after delay
> ...
> com.sun.jersey.api.client.ClientHandlerException: java.net.ConnectException: Connection refused (Connection refused)
> ...
> Caused by: java.net.ConnectException: Connection refused (Connection refused)
> ...
> 

I resolved this by adding `eureka.instance.hostname=localhost` to the discover-service application.properties.

I look forward to reading the next parts of your blog series.

Regards,
Troy

--- a/discovery-service/src/main/resources/application.properties
+++ b/discovery-service/src/main/resources/application.properties
@@ -2,3 +2,5 @@ spring.application.name=discovery-service
 server.port=3000
 eureka.client.registerWithEureka=false
 eureka.client.fetchRegistry=false
+eureka.instance.hostname=localhost
+eureka.client.serviceUrl.defaultZone=http://localhost:${server.port}/eureka/

